contrib/cni/README.md

```diff
@@ -11,6 +11,51 @@ specified by `crio.network.network_dir` in your `crio.conf`).
 In addition, you need to install the [CNI plugins][cni] necessary into
 `/opt/cni/bin` (or the directories specified by `crio.network.plugin_dir`). The
 two plugins necessary for the example CNI configurations are `loopback` and
-`bridge`.
+`bridge`. Below is a tutorial on downloading and setting up the CNI plugins.
 
 [cni]: https://github.com/containernetworking/plugins
+
+### Plugins tutorial
+
+This tutorial will use the latest version of `CNI` plugins from the master branch and build it from source.
+
+Download the `CNI` plugins source tree:
+
+```
+git clone https://github.com/containernetworking/plugins
```

Comment:
Lets refer to a tag that we have tested as working.

Reply:
Won't that get messed up because it's in triple quotes? (you're talking about doing
```
git clone [cni]
```
right)

Reply:
I meant a particular git tag like 0.7.